### DISCUSSION TITLE
retweets with their comments

### DISCUSSION THREAD

**Discussion by MuhammadHadiofficial** [tweepy/tweepy#1389] [Questions] [answered]
I'm using api.retweets('id') function but it only returbns retweets without comments. How can we fetch retweets with comments as well. For example: my tweet had 5 retweets 1 with comment and 4 without comments upon calling function it returned only 4 retweets exluding 1 with comments. Anyway I can fetch all 5 of them with their comment as well.

**Comment by Harmon758** [comment, marked as answer]
This is an API limitation. Since Retweets with comments / Quote Tweets consider the URL to the original Tweet to be part of their text, you can search for them instead, e.g. with a query like `url:1344098443288055808`.
